dist/21-526EZ-ALLCLAIMS-schema.json

```diff
@@ -318,7 +318,7 @@
         "MI",
         "MN",
         "MS",
-        "AK",
+        "MO",
```

Comment:
We are going to only want our 10-10CG changes in this PR.

If these changes were created when you ran build, let's put this in a separate PR. Probably will need to make a branch off of master, run the build, commit the changes, merge that PR before this one.

Reply:
Opened another PR to isolate changes:
https://github.com/department-of-veterans-affairs/vets-json-schema/pull/487

These updates occurred due to me just running `yarn build`